www/js/app.js

```diff
@@ -98,6 +98,15 @@ define(['jquery', 'zimArchiveLoader', 'uiUtil', 'settingsStore','abstractFilesys
     
     // Unique identifier of the article expected to be displayed
     var expectedArticleURLToBeDisplayed = "";
+ 
+    var darkPreference = window.matchMedia("(prefers-color-scheme : dark");
+    darkPreference.addEventListener("change", e=> {
+            if (params.appTheme == "auto") {
```

Comment:
You probably want to set appTheme to `auto_invert` (in `index.html`) rather than `auto`. This will allow it to be extended to support `auto_mwInvert`.